src/clojurewerkz/elastisch/native/conversion.clj

```diff
@@ -812,6 +820,77 @@
       (.searchFrom r ^{:tag "int"} from))
     r))
 
+(defn ->string [text]
+  (if (keyword? text)
+    (name text)
+    (str text)))
+
+(defn attach-suggestion-context [query context]
```

Comment:
The rest of the codebase moves argument vector to the next line. Please follow this style.